cgosqlite/cgosqlite.go

```diff
@@ -131,6 +131,15 @@ func (db *DB) Checkpoint(dbName string, mode sqliteh.Checkpoint) (int, int, erro
 	return int(nLog), int(nCkpt), errCode(res)
 }
 
+func (db *DB) TxnState(schema string) sqliteh.Txn {
```

Comment:
docs? at least to clarify that schema is optional?

Reply:
These methods are documented on the interface they implement. Those link to the sqlite docs, because I don't want to reproduce those here.